rw-system.sh

```diff
@@ -1007,9 +1006,9 @@ if [ "$vndk" -le 30 ];then
 	setprop ro.surface_flinger.use_color_management false
 fi
 
-# Disable ODM display overlay for Realme GT ME, which is annoyingly hard to override
-if getprop ro.boot.prjname |grep -qi 21615;then
-    mount -o bind /system/phh/empty /odm/overlay/android_framework_res_overlay.display.product.21615.apk
+# Disable ODM display overlay for some OPlus devices, which is annoyingly hard to override
+if getprop ro.boot.prjname |grep -qi -e 20846 -e 20847 -e 2084A -e 21615;then
+    mount -o bind /system/phh/empty /odm/overlay/android_framework_res_overlay.display.product.*.apk
```

Comment:
There is no risk that there are multiple overlays?

Reply:
As far as my observations of OPlus devices go. PrjName check and only binding the "fwb display" overlay (i.e. only the notch definitions) should tighten it well enough.